tests/unit/s2n_connection_test.c

```diff
@@ -124,8 +124,25 @@ int main(int argc, char **argv)
      */
     {
         /* Carefully consider any increases to this number. */
-        const uint16_t max_connection_size = 4264;
-        const uint16_t min_connection_size = max_connection_size * 0.9;
+        const uint16_t max_connection_size_64 = 4264;
+        /* this constant determines the min size relative to the max size */
+        const double tolerance = 0.9;
+
+        /* these numbers were obtained with parent commit:
+         * 92b571ae3c71ad3a69511fae65982a93cb9a271b. The size_reduction is
+         * calculated as 64 bit connection size - 32 bit connection size for
+         * some particular feature set.
+         */
+        const uint16_t size_reduction = 4256 - 3260;
```

Comment:
> Would it make sense/be possible to calculate the size reduction based on a fraction of the current 64 bit max size?
My argument is that it does not make sense to treat the 32 bit size difference as a fractional difference. I haven't dug too deeply into this, but my understanding is that the difference between 32 and 64 bit struct sizes is just `number_of_pointers * (8 - 4)`. In which case I think it makes more sense that the difference is relative?

> if there was a separate 32 bit max size const we could manually increase both consts based on however many bytes would be added to both versions?
Do people normally know exactly how much memory they are adding? If the answer is yes then I would agree that this seems like a better approach, but my assumption was that bumping the amount of memory was a rather inexact process, and that this would require experimentally checking 32 bit size anyways.

> maybe the difference between 64 and 32 bit sizes will stay mostly the same.
This was my hope :)

Reply:
Right, okay. I guess we wouldn't know exactly how much to increase both versions by unless we bumped it every time we modified s2n_connection. So maybe your solution makes sense.